production.Dockerfile

```diff
@@ -1,6 +1,9 @@
 # Use the official Ruby image because the Rails images have been deprecated
 FROM ruby:2.5-slim
 
+# Set necessary ENV
+ENV LC_ALL=C.UTF-8
```

Comment:
@pauldoomgov I just added this, I ran into errors using RegExes in ruby without it.